elixir/apps/domain/priv/repo/migrations/20250919205908_add_non_composite_indexes_for_cascade_fks.exs

```diff
@@ -0,0 +1,75 @@
+defmodule Domain.Repo.Migrations.AddNonCompositeIndexesForCascadeFks do
+  use Ecto.Migration
+
+  @disable_ddl_transaction true
+
+  def up do
```

Comment:
For "standard" helpers like the ones here I checked and Ecto actually gets the down queries correct, so we can simplify this a bit.

```suggestion
  def change do
```